data/data/bootstrap/systemd/units/systemd-journal-gatewayd.service.d/certs.conf

```diff
@@ -0,0 +1,6 @@
+[Service]
+ExecStart=
+ExecStart=/usr/lib/systemd/systemd-journal-gatewayd \
```

Comment:
Where is this service defined? Where is it's source.

Reply:
> Where is this service defined? Where is it's source.

The service this is a drop-in for?  I'd guess in a package RHCOS pulls in.

Reply:
I thought this was offered by the kubelet? And it's debug port.

Reply:
This is journal-gatewayd, which is included (at my request) in RHCOS.